requirements.txt

```diff
@@ -1,6 +1,6 @@
 # Runtime requirements for the python 3 version of cirq.
 
-dataclasses; python_version < '3.7'
```

Comment:
Is this change relevant to this PR? I haven't been keeping track on if we've are doing `3.8`.

Reply:
This is PR is just raising the level to 3.7 i.e. < 3.8 should be fine. I've been using cirq on 3.7 for a long time now. The 3.8 work will build on top of this.